CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Thank you for your interest in contributing to the 1Password load-secrets-action project 👋! Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Testing
+
+After following the steps below for signing commits, you can test against your PR with these steps:
+
+1. Create or use an existing repo to run the `load-secrets` GitHub Action
```

Comment:
`[nitpick]`

From what I know steps that contain sentences (like this one) should end with punctuation:
```suggestion
1. Create or use an existing repo to run the `load-secrets` GitHub Action.
```

Same applies to the other steps.